plugin/DirDiff.vim

```diff
@@ -718,7 +736,8 @@ function! <SID>DirDiffOpen()
         echo "There is no diff at the current line!"
     endif
 
-    exec thisWindow.'wincmd w'
+    wincmd k
+    normal gg]c
```

Comment:
Actually I probably followed the existent use of normal on the plugin, as can be seen five lines before. I will change both them.